src/main/java/com/aetherteam/aether_genesis/block/GenesisBlocks.java

```diff
@@ -70,6 +74,7 @@ public static void registerFlammability() {
         fireBlockAccessor.callSetFlammable(GenesisBlocks.STRIPPED_SKYROOT_LOG_WALL.get(), 5, 5);
         fireBlockAccessor.callSetFlammable(GenesisBlocks.SKYROOT_WOOD_WALL.get(), 5, 5);
         fireBlockAccessor.callSetFlammable(GenesisBlocks.STRIPPED_SKYROOT_WOOD_WALL.get(), 5, 5);
+        fireBlockAccessor.callSetFlammable(GenesisBlocks.AETHER_CRAFTING_TABLE.get(), 5, 5);
```

Comment:
Crafting Tables aren't flammable in vanilla

Reply:
sorry